### ISSUE TITLE
error while installing soundscapy with anaconda powershell

### ISSUE THREAD

**Issue by olivier8064** [MitchellAcoustics/Soundscapy#82]
Hi, I tried to install soundscapy but at the end , I encoutered some errors. See on screenshot. 
What to do ? 
Thanks
![Screenshot - 31_08_2024 , 22_36_10](https://github.com/user-attachments/assets/001ee4c8-66a2-40d7-8225-8e4c477d2f92)

**Comment by olivier8064** [comment]
I was wondering with wich Python app do you recommand to install soundscapy ? Spyder doesn't work. Thanks

**Comment by MitchellAcoustics** [comment]
I'm not clear what is going on here - could you please clarify the commands you have used? Is this the result of `pip install soundscapy` within the Anaconda shell? 

To me it looks like the output is saying soundscapy installs okay, but the pyyaml dependency conflicts with a conda dependency. I don't think this can be solved from soundscapy's side. I would recommend updating your anaconda version and trying again.

I would recommend just using your native or typical command prompt (i.e. windows powershell or Mac terminal, etc.).

**Comment by olivier8064** [comment]
Hi, thanks for your reply. Yes it was with "pip install soundscapy" but as you recommand it, I've just tried with windows powershell after have installed python. It works now ! I was able to install soundscapy. What is unclear now for me is where to place my wav sound files. I'll continue to search and open if necessary a new thread. If you have a video explaining how to use soundscapy, it would be very helpfull for me :) Thanks.
